src/components/utils/test/thread_test.cc

```diff
@@ -74,14 +78,18 @@ class ThreadTestDelegate : public threads::ThreadDelegate {
   void WaitForThreadStart() {
     AutoLock test_lock(test_mutex_);
     while (!thread_started_) {
-      cond_var_.Wait(test_lock);
+      cond_var_.WaitFor(test_lock, kMilliSecondsToWait);
```

Comment:
@wolfylambova22 You can check the returned status of WaitFor(). It'll tell you is it because of time-out or not.